interface/usergroup/usergroup_admin_add.php

```diff
@@ -51,7 +52,7 @@
 <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
 <link rel="stylesheet" type="text/css" href="<?php echo $GLOBALS['webroot'] ?>/library/js/fancybox/jquery.fancybox-1.2.6.css" media="screen" />
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/dialog.js"></script>
-<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/js/jquery.1.3.2.js"></script>
+<?php call_required_libraries(array("jquery-min-3-1-1")); ?>
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/js/common.js"></script>
```

Comment:
You can call call_required_libraries(array("jquery-min-3-1-1", "common","fancybox"));
This will call jquery, common.js, fancybox, fancybox.css, as well as dialog.js, as dialog is included automatically.
This will replace lines 53-57.

Reply:
Alright! I'll fix that right now. 


What did you mean by "He has both here..." in the get_fullscreen_pages.php comment? I can't seem to reply to that comment for some reason.